NEWS.rst

```diff
@@ -2,6 +2,50 @@
 
 .. towncrier release notes start
 
+towncrier 21.3.0 (2021-04-02)
+=============================
+
+No significant changes.
```

Comment:
Very minor comment. Feel free to ignore.

Maybe be explicit there are no changes, other than version update.

One might think there might be some change and try to search for them...

```suggestion
No significant changes since the previous release candidate.
```